src/Constants.ts

```diff
@@ -109,7 +109,7 @@ export const CONSTANTS = {
 
   // Also update Documentation/doc/changelog.md when appropriate (when doing a release)
   LatestUpdate: `
-## v2.6.3 Dev: Last updated 15 August 2024
+## v2.7.0 Dev: Last updated 15 August 2024
```

Comment:
Should probably update the date on this line as well.

Reply:
We usually update it when Snarling or d0sboots update the content of changelog.

Reply:
We usually update it when Snarling or d0sboots updates the content of changelog.